Gradle 6.4 execution breaks while trying to change mainClass

Using plugin version 0.2.3 and Gradle 6.4, the following exception is thrown while configuring the `PitestTask`:

```
Caused by: java.lang.IllegalStateException: The value for task ':base:android:pitestDebug' property 'mainClass' is final and cannot be changed any further.
        at org.gradle.api.internal.provider.AbstractProperty$FinalizedValue.beforeMutate(AbstractProperty.java:484)
        at org.gradle.api.internal.provider.AbstractProperty.assertCanMutate(AbstractProperty.java:264)
        at org.gradle.api.internal.provider.AbstractProperty.setSupplier(AbstractProperty.java:213)
        at org.gradle.api.internal.provider.DefaultProperty.set(DefaultProperty.java:69)
        at org.gradle.api.tasks.JavaExec.setMain(JavaExec.java:405)
        at pl.droidsonroids.gradle.pitest.PitestTask_Decorated.setMain(Unknown Source)
        at org.gradle.api.tasks.JavaExec.setMain(JavaExec.java:108)
        at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.setProperty(BeanDynamicObject.java:380)
        at org.gradle.internal.metaobject.BeanDynamicObject.trySetProperty(BeanDynamicObject.java:180)
        at org.gradle.internal.metaobject.CompositeDynamicObject.trySetProperty(CompositeDynamicObject.java:66)
        at org.gradle.internal.metaobject.AbstractDynamicObject.setProperty(AbstractDynamicObject.java:74)
        at pl.droidsonroids.gradle.pitest.PitestTask_Decorated.setProperty(Unknown Source)
        at pl.droidsonroids.gradle.pitest.PitestTask.exec(PitestTask.groovy:311)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
...
```

This is easy to reproduce by bumping the project Gradle version to 6.4 and running `./gradlew funcTest`

--- a/src/main/groovy/pl/droidsonroids/gradle/pitest/PitestTask.groovy
+++ b/src/main/groovy/pl/droidsonroids/gradle/pitest/PitestTask.groovy
@@ -245,6 +245,10 @@ class PitestTask extends JavaExec {
     List<String> overriddenTargetTests  //should be Set<String> or SetProperty but it's not supported in Gradle as of 5.6.1
 
     PitestTask() {
+        //setting during execution doesn't work in 6.4+:
+        //The value for task ':pitest' property 'mainClass' is final and cannot be changed any further.
+        main = "org.pitest.mutationtest.commandline.MutationCoverageReport"
+
         ObjectFactory of = project.objects
 
         testPlugin = of.property(String)
@@ -308,7 +312,6 @@ class PitestTask extends JavaExec {
         //Workaround for compatibility with Gradle <4.0 due to setArgs(List) and setJvmArgs(List) added in Gradle 4.0
         args = createListOfAllArgumentsForPit()
         jvmArgs = ((List<String>)getMainProcessJvmArgs().getOrNull() ?: getJvmArgs())
-        main = "org.pitest.mutationtest.commandline.MutationCoverageReport"
         classpath = getLaunchClasspath()
         super.exec()
     }

